.pre-commit-config.yaml

```diff
@@ -9,7 +9,7 @@ repos:
         stages: [commit]
       - id: check-docstrings
         name: check-docstrings
-        entry: python3 ./stdlib/scripts/check-docstrings.py
+        entry: mojo doc --validate-doc-strings stdlib/src
```

Comment:
Should we remove the python file in this PR or in another one? Is the `check-docstrings.py` managed by copybara?

Reply:
It's managed by copybara so I plan on removing it in a separate PR and also update the contributor docs. Does that work for you?

Reply:
Yep